keyboards/zeix/qwertyqop60hs/info.json

```diff
@@ -0,0 +1,96 @@
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP6","GP5","GP4","GP0","GP12"],
+        "cols": ["GP28","GP27","GP26","GP25","GP24","GP23","GP22","GP3","GP2","GP1","GP8","GP9","GP10","GP11","GP7"]
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "matrix": [0, 0], "x": 0, "y": 0 },
+                { "matrix": [0, 1], "x": 1, "y": 0 },
+                { "matrix": [0, 2], "x": 2, "y": 0 },
+                { "matrix": [0, 3], "x": 3, "y": 0 },
+                { "matrix": [0, 4], "x": 4, "y": 0 },
+                { "matrix": [0, 5], "x": 5, "y": 0 },
+                { "matrix": [0, 6], "x": 6, "y": 0 },
+                { "matrix": [0, 7], "x": 7, "y": 0 },
+                { "matrix": [0, 8], "x": 8, "y": 0 },
+                { "matrix": [0, 9], "x": 9, "y": 0 },
+                { "matrix": [0, 10], "x": 10, "y": 0 },
+                { "matrix": [0, 11], "x": 11, "y": 0 },
+                { "matrix": [0, 12], "x": 12, "y": 0 },
+                { "matrix": [0, 13], "x": 13, "y": 0 },
+                { "matrix": [0, 14], "x": 14, "y": 0 },
+                { "matrix": [1, 0], "w": 1.5, "x": 0, "y": 1 },
+                { "matrix": [1, 1], "x": 1.5, "y": 1 },
+                { "matrix": [1, 2], "x": 2.5, "y": 1 },
+                { "matrix": [1, 3], "x": 3.5, "y": 1 },
+                { "matrix": [1, 4], "x": 4.5, "y": 1 },
+                { "matrix": [1, 5], "x": 5.5, "y": 1 },
+                { "matrix": [1, 6], "x": 6.5, "y": 1 },
+                { "matrix": [1, 7], "x": 7.5, "y": 1 },
+                { "matrix": [1, 8], "x": 8.5, "y": 1 },
+                { "matrix": [1, 9], "x": 9.5, "y": 1 },
+                { "matrix": [1, 10], "x": 10.5, "y": 1 },
+                { "matrix": [1, 11], "x": 11.5, "y": 1 },
+                { "matrix": [1, 12], "x": 12.5, "y": 1 },
+                { "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
+                { "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
+                { "matrix": [2, 1], "x": 1.75, "y": 2 },
+                { "matrix": [2, 2], "x": 2.75, "y": 2 },
+                { "matrix": [2, 3], "x": 3.75, "y": 2 },
+                { "matrix": [2, 4], "x": 4.75, "y": 2 },
+                { "matrix": [2, 5], "x": 5.75, "y": 2 },
+                { "matrix": [2, 6], "x": 6.75, "y": 2 },
+                { "matrix": [2, 7], "x": 7.75, "y": 2 },
+                { "matrix": [2, 8], "x": 8.75, "y": 2 },
+                { "matrix": [2, 9], "x": 9.75, "y": 2 },
+                { "matrix": [2, 10], "x": 10.75, "y": 2 },
+                { "matrix": [2, 11], "x": 11.75, "y": 2 },
+                { "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
+                { "matrix": [2, 12], "x": 12.75, "y": 2 },
```

Comment:
looking at the image provided in `readme.md`, a position for NUHS doesn't exist. Is this true to the actual PCB?

Reply:
This is indeed correct - I will update the matrix info.json to reflect it.

Reply:
Good catch! I've removed the unused reference.